less/home-page.less

```diff
@@ -1,3 +1,20 @@
+// Title Navigation Content
+
+.title {
+  padding-bottom: 2vh;
+  background-color: @wild-sand;
+  .flex();
+  flex-direction: column;
+  align-items: center;
+  &> * {
```

Comment:
oooo... I don't think I have seen the * selector used like this. I like the idea.

Reply:
Yeah that was a gamble and it may come with a few gotchas, but it worked in this use-case!